tests/perf-regression-write.60m.100threads.10M-keys.rf-3.yaml

```diff
@@ -24,20 +25,33 @@ backends: !mux
         gce_instance_type_loader: 'n1-standard-2'
         gce_root_disk_type_loader: 'pd-standard'
         gce_n_local_ssd_disk_loader: 0
-        gce_instance_type_monitor: 'n1-standard-1'
+        gce_instance_type_monitor: 'n1-standard-2'
         gce_root_disk_type_monitor: 'pd-standard'
         gce_root_disk_size_monitor: 50
         gce_n_local_ssd_disk_monitor: 0
-        scylla_repo: 'https://s3.amazonaws.com/downloads.scylladb.com/rpm/unstable/centos/branch-1.7/17/scylla.repo'
+        scylla_repo: 'REPO_FILE_PATH'
         us_east_1:
           gce_datacenter: 'us-east1-b'
 
-
+    aws: !mux
+        cluster_backend: 'aws'
+        instance_type_loader: 'c4.large'
+        instance_type_monitor: t2.small
+        instance_type_db: 'i2.2xlarge'
+        us_east_1:
+            user_credentials_path: ''
+            region_name: 'us-east-1'
+            security_group_ids: 'sg-c5e1f7a0'
+            subnet_id: 'subnet-ec4a72c4'
+            ami_db_scylla_user: 'centos'
+            ami_loader_user: 'centos'
+            ami_monitor_user: 'centos'
+            ami_id_db_scylla: 'AMI_ID'
```

Comment:
What's the recommended automatic way to replace the string with real parameter?

- sed ?

Can we also add a `$` prefix to avoid the wrong replacement?

Reply:
I don't know what you mean by recommended, but as you know In Jenkins we use sed...

>Can we also add a $ prefix to avoid the wrong replacement?
Not sure what do you mean by wrong replacement... to change it to: $AMI_ID?